SITCOMTN-076.tex

```diff
@@ -49,6 +53,325 @@
+
+%In general, it is NOT permitted to copy/move data products from Rubin Observatory computing facilities to other locations without explicit permission.
+%See Rubin Observatory Data Security Standards Implementation (DMTN-199).
+
+\subsection{Technotes}
+\label{technotes}
+
+\href{https://developer.lsst.io/project-docs/technotes.html}{Technotes} are a way for Rubin Observatory team members to write standalone documents that are native to the web, can be cited in literature, and are easy to write, publish, and update.
+A listing of technotes is available at \url{www.lsst.io}.
+
+During the on-sky commissioning period with LSSTCam, technotes are anticipated to be one of the primary mechanisms to share information about Rubin Observatory data with the science community.
+Project team members are encouraged to document their analyses in the form of technotes, e.g., to describe an algorithm or analysis software, to characterize and/or propose a solution to an outstanding issue, or to present the results of a science verification / validation study.
+Technotes may present science validation analyses, but are not intended to include novel scientific results / discoveries.
+Each technote should have a well defined scope.
+
+Prior to the associated Early Science release, technotes that involve proprietary on-sky LSSTCam data must be first drafted in a restricted space and reviewed/approved by the Project using a standard checklist in a timely manner before posting.
+This process will ensure that released technotes meet basic standards for documentation quality and conform to Rubin Observatory information release policies.
+As living documents, technotes may describe work in progress and may be updated as studies progress.
+Once released, updated technote versions must maintain compliance with the approval checklist criteria.
+
+Derived data products may be documented and shared via technotes that have been approved for public release.
+
+%\section{Frequently Asked Questions}
+
+\subsection{Presentations}
+
+When presenting on Rubin Observatory technical and scientific performance at institutional meetings, LSST Science Collaboration meetings, scientific conferences, seminars and colloquia, etc., Project team members are responsible for following presentation guidelines adopted by the Project.
+The presentation content related to Rubin Observatory technical and scientific performance may only include released materials.
+%Approved derived data products may be openly discussed.
+%When presenting on Rubin Observatory at
+%\href{https://www.lsst.org/scientists/codes-of-conduct}{professional standards of conduct} adopted by the Project.
+
+\subsection{External Proposals}
+
+Members of the Project team are welcome to include descriptions of their commissioning activities to support external grant proposals and observing proposals.
+Embargoed information cannot be included, except with specific approval of the Rubin Observatory Project Office.
```

Comment:
I think we'll need to specify an approval or "how to seek permission" process

Reply:
I agree. I am not sure that we are prepared to provide that detail at this time.